### PR TITLE
Move github actions to root directory

### PR DESCRIPTION
**Pull request by clemens-msupply**
The reason it was failing was that I edited the wrong workflow file. I also moved the workflow files from the server over to the root directory (untested so far).

### CODE REVIEW

File: client/.github/workflows/comment.md [hunk 1]
```diff
@@ -1,6 +0,0 @@
-## Bundle size difference
```
Comment (mark-prins) [inline client/.github/workflows/comment.md:1]: I thought that this file was required by the bundle size action?
```
- name: Comment
        uses: NejcZdovc/comment-pr@v1
        with:
          file: "comment.md"
```

Comment (clemens-msupply) [reply client/.github/workflows/comment.md:1]: The same file was already in `.github/workflows/comment.md` thats why it doesn't show up in the PR

Comment (mark-prins) [reply client/.github/workflows/comment.md:1]: ah. oops

**Review by mark-prins — APPROVED** [review]
👍

**Pull request merged**